hypothesis-python/tests/nocover/test_sampled_from.py

```diff
@@ -70,6 +70,7 @@ def test_unsat_sets_of_samples(x):
     assert False
 
 
+@settings(suppress_health_check=[HealthCheck.too_slow, HealthCheck.filter_too_much])
```

Comment:
I'm not really comfortable with this since it seems to defeat the purpose of the test, but after merging #2031 and rebasing I don't think it will be needed.

Reply:
Yup, agreed. I think it's not totally defeating the point of the test, but it definitely weakens it. Either way, it's no longer necessary.